CircleSpawn/CircleSpawn/CircleSpawnController.swift

```diff
@@ -1,11 +1,67 @@
 import UIKit
 
-class CircleSpawnController: UIViewController {
+class CircleSpawnController: UIViewController, UIGestureRecognizerDelegate {
+    
+    override func loadView() {
+        view = UIView()
+        view.backgroundColor = .white
+    }
+    
+    var locationInSubView = CGPoint()
```

Comment:
Po to mamy w Swifcie optionale, żeby ich używać. Teraz ten punkt ma domyślną wartość `(x: 0.0, y: 0.0)`, ale tak naprawdę to powinien być `nil`.

W takich przypadkach używamy `CGPoint?`, bo ten punkt nie ma wartości.